tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/BatchRequestBuilderTests.cs

```diff
@@ -2,6 +2,7 @@
 //  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
 // ------------------------------------------------------------------------------
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
which member is obsolete? could we move this suppression to just the member that's causing a problem so it does not mask potential future obsolescence?

Reply:
Moved this for this specific file. 
In general the BatchRequests are marked obsolete to encourage use of the BatchCollection types. So, the test files touching those are easier marked for the whole file give that all the APIs are being redirected to the new APIs